docs/authorization/gare.rst

```diff
@@ -1,29 +1,29 @@
 Auth Requirements Errors
 ========================
 
-Globus Auth Requirements Error is a response format that conveys to a client any
+'Globus Auth Requirements Error' is a response format that conveys to a client any
```

Comment:
I can't comment on the heading, which is outside of this change, but perhaps worth revising to "Globus Auth Requirements Errors (GAREs)"? I don't feel super strongly, but it means 'GARE' will show up in the navigation, which I think is a usability win.

Reply:
I'm happy to include this -- I needed to touch this file anyway, due to the function renames. I'll include with the other feedback updates.